### PR TITLE
fix: math/fibonacci linter warnings.

### PR DESCRIPTION
**Pull request by fhlasek**
#### Description of Change
as title

#### Checklist

- [X] Added description of change
- [X] Added file name matches [File name guidelines](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#New-File-Name-guidelines)
- [X] Added tests and example, test must pass
- [X] Added documentation so that the program is self-explanatory and educational - [Doxygen guidelines](https://www.doxygen.nl/manual/docblocks.html)
- [X] Relevant documentation/comments is changed or added
- [X] PR title follows semantic [commit guidelines](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#Commit-Guidelines)
- [X] Search previous suggestions before making a new one, as yours may be a duplicate.
- [X] I acknowledge that all my contributions will be made under the project's license.

Notes: 

<a href="https://gitpod.io/#https://github.com/TheAlgorithms/C-Plus-Plus/pull/1047"></a>

### CODE REVIEW

File: math/fibonacci.cpp [hunk 1]
```diff
@@ -17,8 +17,9 @@
 unsigned int fibonacci(unsigned int n) {
```
Comment (kvedala) [inline math/fibonacci.cpp:17]: shall we take this opportunity to update the doc for this function by adding `@param` and `@returns`?

Comment (fhlasek) [reply math/fibonacci.cpp:19]: done

File: math/fibonacci.cpp [hunk 2]
```diff
@@ -13,12 +13,15 @@
 
 /**
  * Recursively compute sequences
+ * @param n input
+ * @returns n-th element of the Fbinacci's sequence
  */
 unsigned int fibonacci(unsigned int n) {
```
Comment (kvedala) [inline math/fibonacci.cpp:19]: could you update the code to use `uint64_t` instead of `int` and `unsigned int`.
Thanks

Comment (fhlasek) [reply math/fibonacci.cpp]: done

**Pull request merged**